core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -56,6 +56,8 @@
    */
   public static final String CARBON_INPUT_SEGMENTS = "carbon.input.segments.";
 
+  public static final String VALIDATE_CARBON_INPUT_SEGMENTS = "validate.carbon.input.segments.";
```

Comment:
What is the use of this property? why is it only required in case of an aggregate table?

Reply:
This property is will decide whether the segments have to be looked up and validated or not.
For incremental load into pre-aggregate table as only the last segment would be loaded therefore we dont need to lookup or validate the segment that has been set.

Reply:
Please add the comment of usage here

Reply:
Added comment